Automattic/wp-calypso
# My Home: Add flag to experiment with new layout

Requires D41677-code

#### Changes proposed in this Pull Request

This PR adds a `home/experimental-layout` flag to experiment with the new views we added for the improved Home layout in D41677-code without breaking the current behavior in production.

The flag is disabled in all environments (in order to don't block the local environment for developers visiting My Home), so it's need to be enabled by explicitly visiting My Home with the `flags=home/experimental-layout` query arg. 

This flag will allow us to ship next iterations as small as possible, making changes easier to review. 
On this first PR we are only defining the new locations described in pbAPfg-g0-p2: notices, primary, secondary and tertiary. Styles have not been adjusted in order to keep this PR as small as possible.

Once these changes land we can continue porting commits we initially pushed to #41000 as separated PRs:
- Adjust styles for quick links: 6395378708465cde3bea188706921cebce88b9ab
- Combine educational cards: b806c106fae44beb4725b7d8c479c14004f147c1

#### Testing instructions

- Apply D41677-code and sandbox the API.
- Go to `/home/:site` with a prelaunch, launched, established and WP for Teams site and observe the layout. It should be the same as production.
- Enable the flag by going to `/home/:site?flags=home/experimental-layout` and observe the layout again. You should see the new locations described in pbAPfg-g0-p2: notices, primary, secondary and tertiary.
![image](https://user-images.githubusercontent.com/1233880/79356969-ba17a380-7f3f-11ea-9315-2c4e2ac52077.png)
